src/Core/Logger/Factory/Logger.php

```diff
@@ -51,14 +51,14 @@ class Logger
 	 *
 	 * @var string[]
 	 */
-	private static $ignoreClassList = [
+	public static $ignoreClassList = [
```

Comment:
Uff, public attributes should be ALWAYS avoided according some good code practices.

Reply:
This looks less of an attribute than a class constant, which would make it public and immutable.
```suggestion
	const IGNORE_CLASS_LIST = [
```
Don't forget to update lines 87 and 203 in this file that I can't review.

Reply:
> This looks less of an attribute than a class constant, which would make it public and immutable.
> 
> Don't forget to update lines 87 and 203 in this file that I can't review.

Okay, then the `$fooBar` notation has confused me into believing that `$ignoreClassList` is a class field/attribute instead of a static immutable.

Reply:
You comment was good, my comment was a change suggestion.

Reply:
Okay, no big deal. :-)